client/src/components/Attachment/UploadAttachment.js

```diff
@@ -88,27 +106,39 @@ const UploadAttachment = ({ getRelatedObject, edit, saveRelatedObject }) => {
             toast.error(
               `Attachment content upload failed for ${
                 selectedAttachment.fileName
-              }: ${error.response?.data?.error || error.message}`
+              }: ${error.response?.data?.error || error.message}`,
+              {
+                autoClose: false,
+                closeOnClick: true
+              }
             )
           })
       })
       .catch(error =>
         toast.error(
-          `Attachment upload for ${selectedAttachment.fileName} failed: ${error.message}`
+          `Attachment upload for ${selectedAttachment.fileName} failed: ${error.message}`,
+          {
+            autoClose: false,
+            closeOnClick: true
+          }
         )
       )
   }
 
   const handleFileEvent = async e => {
     // Control related object has an uuid or not
     if (!relatedObject.uuid) {
-      saveRelatedObject().then(async response => {
+      try {
+        const response = await saveRelatedObject()
         relatedObject.uuid = response
-        await attachmentSave(e)
-      })
-    } else {
-      await attachmentSave(e)
+      } catch (error) {
+        toast.error(`Attachment content upload failed: ${error.message}`, {
+          autoClose: false,
+          closeOnClick: true
+        })
+      }
     }
+    await attachmentSave(e)
```

Comment:
Error not handled:

![image](https://github.com/NCI-Agency/anet/assets/45197818/07cb21f5-8406-4a9c-a6e8-158793c2ea4d)

Reply:
This error happened cause, there is no control for null default classifiation for updateAttachment query so I added control its okay now.